### ISSUE TITLE
Have the documentation always point at the latest release (stable tag vs latest github release)

### ISSUE THREAD

**Issue by kostis-codefresh** [argoproj/argo-rollouts#1262]
At the time of writing even though version 1.0.1 is released, the "stable" Git tag still points to 0.10.2.

This creates confusion for users since the documentation page at https://argoproj.github.io/argo-rollouts/installation.html points to installation manifest at https://raw.githubusercontent.com/argoproj/argo-rollouts/stable/manifests/install.yaml

This means that somebody who is following the documentation will install version 0.10.2 which is probably not their intent.

We should change that and I see two possible options.

1. Have the "stable" tag automatically updated when a new release tag is updated
2. Simply change the documentation to mention https://github.com/argoproj/argo-rollouts/releases/latest/download/install.yaml so that users will always get the latest Github release regardless of what the stable tag is doing (I would think that we don't even need a stable tag any more, if it was created for this purpose)

I am happy to submit a PR for the second scenario if that is the route we want to take.

---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritize the issues with the most 👍.

**Comment by jessesuen** [comment]
Yes, you're right. I think option 2 is the right move at this point since we no longer commit the install manifests to git.
